packages/core/src/middleware/koa-security-headers.ts

```diff
@@ -49,15 +49,21 @@ export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(
   /** Google Sign-In (GSI) origin for Google One Tap. */
   const gsiOrigin = 'https://accounts.google.com/gsi/';
 
-  // We have the following use cases:
-  //
-  // - We use `react-monaco-editor` for code editing in the admin console. It loads the monaco
-  // editor asynchronously from jsDelivr.
-  //
-  // Allow the CDN src in the CSP.
-  // Allow blob: for monaco editor to load worker scripts
+  /** We have the following use cases:
+   *
+   * - We use `react-monaco-editor` for code editing in the admin console. It loads the monaco
+   * editor asynchronously from jsDelivr.
+   *
+   * - Allow the CDN src in the CSP.
+   * - Allow blob: for monaco editor to load worker scripts
+   */
   const cdnSources = ['https://cdn.jsdelivr.net/', 'blob:'];
 
+  /**
+   * Third-party API endpoints
+   */
+  const apiSources = ['https://api.inkeep.com/', 'https://api.io.inkeep.com/'];
```

Comment:
This can be added in the cloud code only.

Reply:
Removed from this PR. Thx